packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -121,6 +121,72 @@ const writeRecords = async ({
   });
 };
 
+function extractCollection(cumulusMessage) {
```

Comment:
Couple of comments: 

Docstrings with typing please https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=292913776 are needed in plain JS. 

That said, just use ?. in the main method and remove all these.  

(also we should be always using  const = () => {} as the function syntax and avoid `function foo`, as const prevents accidental mutation (also for style consistency / format reasons, generally, although that's a change to be made wholesale here).

Reply:
I forgot about the ? syntax, much cleaner